test/integration/__snapshots__/integration.test.ts.snap

```diff
@@ -294,8 +294,8 @@ export interface UsersInput {
   jsonbArrayCol?: Json[] | null;
   timestamptzArrayCol?: Date[] | null;
 }
-const Users = {
-  tableName: 'Users',
```

Comment:
RE my change here: https://github.com/danvk/pg-to-ts/pull/14/files/272aacaf8d5cba9a120ff44d10e89e21cd590f7f#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R104


This change I made corrects this. The camelCase option is meant to apply camelCase to the column names only, right?

Reply:
The description of the camelCase option is: `Camel-case columns (e.g. user_id --> userId)`

Reply:
I think it makes sense to only make the column names camelCase (like it says in the description of the camelCase option).